src/QmlControls/ScreenTools.qml

```diff
@@ -107,7 +108,10 @@ Item {
         property real   fontWidth:    contentWidth
         property real   fontHeight:   contentHeight
         Component.onCompleted: {
-            var baseSize = QGroundControl.baseFontPointSize;
+            var baseSize = QGroundControl.corePlugin.options.defaultFontPointSize
```

Comment:
I still think this would be better done through the ability for a plugin to override any setting. That way it requires no code change in the mainline code.